sql/size_utils.sql

```diff
@@ -67,46 +67,25 @@ RETURNS TABLE (
 AS '@MODULE_PATHNAME@', 'ts_dist_remote_chunk_info' LANGUAGE C VOLATILE STRICT;
 
 CREATE OR REPLACE FUNCTION _timescaledb_internal.hypertable_local_size(
-    schema_name_in name,
-    table_name_in name)
+	schema_name_in name,
+	table_name_in name)
 RETURNS TABLE (
-    table_bytes bigint,
-    index_bytes bigint,
-    toast_bytes bigint,
-    total_bytes bigint)
-LANGUAGE PLPGSQL VOLATILE STRICT AS
+	table_bytes bigint,
+	index_bytes bigint,
+	toast_bytes bigint,
+	total_bytes bigint)
+LANGUAGE SQL VOLATILE STRICT AS
 $BODY$
-BEGIN
-   RETURN QUERY 
-   SELECT
-      (sub2.table_bytes + sub2.compressed_heap_bytes)::bigint as heap_bytes,
-      (sub2.index_bytes + sub2.compressed_index_bytes)::bigint as index_bytes,
-      (sub2.toast_bytes + sub2.compressed_toast_bytes)::bigint as toast_bytes,
-      (sub2.total_bytes + sub2.compressed_heap_bytes + sub2.compressed_index_bytes + sub2.compressed_toast_bytes)::bigint as total_bytes 
-   FROM
-      (
-         SELECT
-            *,
-            sub1.total_bytes - sub1.index_bytes - sub1.toast_bytes AS table_bytes 
-         FROM
-            (
-               SELECT
-                  sum(ch.total_bytes) as total_bytes,
-                  COALESCE( sum(ch.index_bytes) , 0 ) as index_bytes,
-                  COALESCE( sum(ch.toast_bytes), 0 ) as toast_bytes,
-                  COALESCE( sum(ch.compressed_heap_size) , 0 ) as compressed_heap_bytes,
-                  COALESCE( sum(ch.compressed_index_size) , 0) as compressed_index_bytes,
-                  COALESCE( sum(ch.compressed_toast_size) , 0 ) as compressed_toast_bytes 
-               FROM
-                  _timescaledb_internal.hypertable_chunk_local_size ch 
-               WHERE
-                  schema_name = schema_name_in 
-                  AND table_name = table_name_in 
-               GROUP BY
-                  hypertable_id 
-            ) sub1 
-      ) sub2;
-END;
+	SELECT
+		(COALESCE(sum(ch.total_bytes), 0) - COALESCE(sum(ch.index_bytes), 0) - COALESCE(sum(ch.toast_bytes), 0) + COALESCE(sum(ch.compressed_heap_size), 0))::bigint AS heap_bytes,
```

Comment:
Nit: you capitalize `COALESCE` but not `sum`. Shouldn't we handle these the same?

Reply:
This was the way it was before these changes. The code has larger issues with formatting and inconsistencies. There was an agreement to fix this is a separate PR since it requires running the code through a SQL formatting tool.